assets/built-in/transformers/dockerfile/dockerimagebuildscript/templates/buildandpushimages_multiarch.sh

```diff
@@ -36,13 +36,28 @@ fi
 if [ "$#" -eq 3 ]; then
   PLATFORMS=$3
 fi
+
+CONTAINER_RUNTIME={{ .ContainerRuntime }}
```

Comment:
Similar to the `buildimages.sh` and `pushimages.sh` file, we should now also allow the user to choose the container runtime while executing the `buildandpushimages_multiarch.sh` file.
 
https://github.com/konveyor/move2kube/blob/565394bf94dcbab7ea340b7daf603f8dee326a14/assets/built-in/transformers/dockerfile/dockerimagebuildscript/templates/buildimages.sh#L19
 
https://github.com/konveyor/move2kube/blob/565394bf94dcbab7ea340b7daf603f8dee326a14/assets/built-in/transformers/dockerfile/dockerimagebuildscript/templates/buildimages.sh#L27
 
https://github.com/konveyor/move2kube/blob/565394bf94dcbab7ea340b7daf603f8dee326a14/assets/built-in/transformers/containerimagespushscript/templates/pushimages.sh#L20
 
https://github.com/konveyor/move2kube/blob/565394bf94dcbab7ea340b7daf603f8dee326a14/assets/built-in/transformers/containerimagespushscript/templates/pushimages.sh#L30

Reply:
Here also we can add an example on how to specify the container runtime (along with registry namespace/registry url/platforms) while executing the file.

https://github.com/konveyor/move2kube/blob/565394bf94dcbab7ea340b7daf603f8dee326a14/assets/built-in/transformers/dockerfile/dockerimagebuildscript/templates/buildandpushimages_multiarch.sh#L20